src/test/java/com/google/api/codegen/testdata/ruby/ruby_doc_library.baseline

```diff
@@ -2666,118 +2678,146 @@ module Library
 
```

Comment:
nit: I wonder if it's idiomatic to do avoid this repetition using functional practices (rough example shown below).

```rb 
create_api_call = Google::Gax.instance_method(:create_api_call)
if exception_transformer
  create_api_call = lambda do |stub, defaults|
    Google::Gax.instance_method(:create_api_call).call(stub, defaults, exception_transformer)
  end
end
@create_shelf = create_api_call.call(
  @library_service_stub.method(:create_shelf),
  defaults["create_shelf"]
)

Reply:
I'm new to Ruby so it's difficult for me to answer this. However, I would generally avoid this because it adds a new code path and (more importantly) I think it's quite a bit more difficult to read.

Reply:
Agreed!